application/dash_application/utility/log_utils.py

```diff
@@ -0,0 +1,30 @@
+from collections import Counter
+from typing import List
+
+import datetime
+import logging
+import json
+import pdb
+
+compare = lambda s, t: Counter(s) == Counter(t)
+
+def collapse_if_all_selected(selected_items: List[str], all_items: List[str], all_title: str) -> List[str]:
+    if compare(selected_items, all_items):
+        return [all_title] # Why array?
```

Comment:
I put it in as an array because `selected_items` is an array, and figured it might be easier to manipulate these in logging if they're all of the same type. You're more experienced in the logging manipulation than I am though -- what do you think?

Reply:
Updated the comment to reflect the decision to return an array for the sake of format consistency